Project.toml

```diff
@@ -4,18 +4,16 @@ repo = "https://github.com/JuliaPolyhedra/LRSLib.jl.git"
 version = "0.5.0"
 
 [deps]
-BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
-Homebrew = "d9be37ee-ecc9-5288-90f1-b9ca67657a75"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Polyhedra = "67491407-f73d-577b-9b50-8179a7c68029"
+lrslib_jll = "3873f7d0-7b7c-52c3-bdf4-8ab39b8f337a"
 
 [compat]
-BinDeps = "0.7, 0.8, 1"
-Homebrew = "0.7"
-Polyhedra = "0.6.8"
-julia = "1"
+Polyhedra = "0.6.14"
+julia = "1.3"
+lrslib_jll = "0.2.0"
```

Comment:
This would allow `0.2.0`, maybe we should do `"= 0.2.0"`, see https://pkgdocs.julialang.org/v1/compatibility/#Equality-specifier

Reply:
Oh whoops.  I think what we want is `"0.2"`, that is, any version in the range `[0.2.0, 1.0.0)`?  IIUC, that is the range of versions that have to be backward-compatible with 0.2.0.

Reply:
Yes but jll packages don't really respect that in my experience. We can try and see but it's safer to just fix and update for each new update of the jll. It's not like the jll will be updated so often.

Reply:
yeah, fair enough
```suggestion
lrslib_jll = "= 0.2.0"
```